src/components/designer/taro/actions/MintAssetModal.spec.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/dom';
+import { waitFor } from '@testing-library/react';
+import { Status } from 'shared/types';
+import { BitcoindLibrary } from 'types';
+
+import {
+  injections,
+  lightningServiceMock,
+  taroServiceMock,
+  renderWithProviders,
+} from 'utils/tests';
+
+import { createNetwork, createTarodNetworkNode, getImageCommand } from 'utils/network';
+import { testRepoState, testManagedImages } from 'utils/tests';
+
+import MintAssetModal from './MintAssetModal';
+
+const bitcoindServiceMock = injections.bitcoindService as jest.Mocked<BitcoindLibrary>;
+
+testManagedImages[0].version = testRepoState.images.LND.latest;
```

Comment:
it is set tp defaultRepoState.images.LND.latest which is currently '0.15.5-beta'

Reply:
Ah I see now. I really don't like the code smell of mutating global vars in the outer scope. I think it would be better to add the taro node via `store.network.addNode()` in the tests instead. There, you would be able to specify which versions to use. So you create the network with just a bitcoind node, then in `beforeEach()` you can add LND and Tarod nodes. This also has the benefit of not needing to manually create all of the initialState you currently have.

Reply:
This is going to take me some time to wrap my head around. But I think I'm on the right track

Reply:
Done, but I feel I could have done it better.